AccountingService/Interface/ITimeService.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace AccountingService.Interface
+{
+    public interface ITimeService
+    {
+        int Minutes { get; }
```

Comment:
This is not needed here. You can move it to configuration alongside with audience, issuer and secret.